references/cicd-sharedflow-pipeline/test/integration/test/integration/apikey.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Get Currency Rates Feature
+	As an API user
+	I want to make sure the API is protected by API Key
+	So that the API is secured
+
+	Scenario: Send request without API Key
+	    When I GET /
+	    And  response body should contain Unauthorized
+	    And response code should be 401
+
+	Scenario: Send request with dummy API Key
+	    Given I set a dummy apikey header
+	    When I GET /
+	    And  response body should contain Unauthorized
+	    And response code should be 401
+
+	Scenario: Send request with correct API Key
+	    Given I set a correct apikey header
```

Comment:
this can use a default step with... 
```
Given I set apikey header to `APIKEY`
```

and set APIKEY in init.js

see <https://github.com/apickli/apickli#variable-injection> for more

Reply:
Done